websites/S/SPWN/presence.ts

```diff
@@ -0,0 +1,148 @@
+import { ActivityType, Assets, getTimestampsFromMedia } from 'premid'
+
+const presence = new Presence({
+  clientId: '1417881802193764492',
+})
+const browsingTimestamp = Math.floor(Date.now() / 1000)
+
+enum ActivityAssets { // Other default assets can be found at index.d.ts
+  Logo = 'https://i.imgur.com/LvbXO0a.png',
+}
+
+presence.on('UpdateData', async () => {
+  const strings = await presence.getStrings({
+    playing: 'general.playing',
+    paused: 'general.paused',
+    watchingLive: 'general.watchingLive',
+    buttonViewPage: 'general.buttonViewPage',
+    viewAPage: 'general.viewAPage',
+  })
+
+  const [privacy, buttons] = await Promise.all([
+    presence.getSetting<boolean>('privacy'),
+    presence.getSetting<boolean>('buttons'),
+  ])
+
+  const presenceData: PresenceData = {
+    type: ActivityType.Watching,
+    largeImageKey: ActivityAssets.Logo,
+    startTimestamp: browsingTimestamp,
+    smallImageKey: Assets.Play,
+  }
+  const { pathname } = document.location
+
+  if (pathname.startsWith('/_')) {
+    // internal site (e.g. firebase auth)
+    presence.clearActivity()
+    return
+  }
+
+  const video = document.querySelector<HTMLVideoElement>('video')
+
+  const eventId = /\/events\/([^/]+)/.exec(document.location.pathname)?.[1]
+  if (eventId) {
+    if (pathname.endsWith('/streaming')) {
+      presenceData.details = strings.watchingLive
+
+      if (privacy === false) {
+        const title = document.querySelector<HTMLParagraphElement>(`#Streaming > div:nth-of-type(3) > p`)?.textContent
+        if (title) {
+          presenceData.state = title
+          presenceData.largeImageText = title
+        }
+
+        const poster = getPoster()
+        if (poster) {
+          presenceData.largeImageKey = poster
+        }
+      }
+    }
+    else {
+      if (privacy === false) {
+        const title = document.querySelector<HTMLHeadingElement>(`h2`)?.textContent
```

Comment:
Use `'` instead of "`" for static strings like these
```suggestion
        const title = document.querySelector<HTMLHeadingElement>('h2')?.textContent
```